docs/source/configuration/config_file.rst

```diff
@@ -73,6 +73,16 @@ Source paths can be configured via ``src`` parameter. If the path does not exist
 If you don't provide source paths in the cli, they will be taken from the closest configuration file. In a setup with
 multiple configuration files, the source paths from other configurations than the closest will be ignored.
 
+``.robotidy`` file uses the same toml syntax as ``pyproject.toml`` file but allows to skip ``tool.robotodiy`` section:
```

Comment:
```suggestion
``.robotidy`` file uses the same toml syntax as ``pyproject.toml`` file but allows to skip ``tool.robotidy`` section:
```

Reply:
So many way to insert typo into this name :D

Reply:
Yup :D I had similar case at work with word "relevancy". It was written in three different ways in one file :D All incorrect :D